partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/PlanTyperTestsPorted.kt

```diff
@@ -3827,15 +3821,26 @@ class PlanTyperTestsPorted {
                 name = "Pets should not be accessible #1",
                 query = "SELECT * FROM pets",
                 expected = BagType(
-                    StructType(
-                        fields = emptyMap(),
-                        contentClosed = true,
-                        constraints = setOf(
-                            TupleConstraint.Open(false),
-                            TupleConstraint.UniqueAttrs(true),
-                            TupleConstraint.Ordered,
-                        )
-                    ),
+                    unionOf(
+                        StructType(
+                            fields = emptyMap(),
+                            contentClosed = false,
+                            constraints = setOf(
+                                TupleConstraint.Open(true),
+                                TupleConstraint.UniqueAttrs(false),
+                            )
+                        ),
+                        StructType(
+                            fields = listOf(
+                                StructType.Field("_1", ANY)
+                            ),
+                            contentClosed = true,
+                            constraints = setOf(
+                                TupleConstraint.Open(false),
+                                TupleConstraint.UniqueAttrs(true),
+                            )
+                        ),
```

Comment:
So why are there two possible struct types returned for this test and a couple below?
- one that is open content and empty set of fields
- one that is closed content w/ one field, `_1` of type `ANY`